text/0000-variadic-generics.md

```diff
@@ -195,6 +195,15 @@ However, this approach introduces a lot of additional complexity. One of the
 complications is that such a trait couldn't be implemented for `()`, so
 there would have to be separate `Cons` and `Split` traits, rather than one
 unified `Tuple`.
+- Allow partial borrows or tuple reference splitting. Previous proposals have
+included a transformation from `&(Head, Tail...)` to `(&Head, &Tail...)`.
+However, this would require that every tuple `(Head, Tail...)` actually
+contain a sub-tuple `tail...`. This would get in the way of desirable
```

Comment:
Right, sorry, I meant `(Head, Tail)` or `(Tail, Head)`, but it's have to *always apply* even when variadics are used nowhere.